src/swc/aeon/io/api.py

```diff
@@ -32,38 +32,47 @@ def aeon(seconds: float | pd.Index | pd.Series) -> datetime.datetime | pd.Index
 @overload
 def to_datetime(seconds: float) -> datetime.datetime: ...
 @overload
-def to_datetime(seconds: pd.Series) -> pd.Series: ...
+def to_datetime(seconds: pd.Index) -> pd.DatetimeIndex: ...
 @overload
-def to_datetime(seconds: pd.Index) -> pd.Index: ...
-def to_datetime(seconds: float | pd.Index | pd.Series) -> datetime.datetime | pd.Index | pd.Series:
-    """Converts a Harp timestamp, in seconds, to a datetime object.
+def to_datetime(seconds: pd.Series) -> pd.Series: ...
+def to_datetime(seconds: float | pd.Index | pd.Series) -> datetime.datetime | pd.DatetimeIndex | pd.Series:
+    """Converts a UTC Harp timestamp to datetime.
+
+    This function converts a scalar, Index or Series into a datetime type.
 
     Args:
-        seconds (float, int, pandas.Series): The Harp timestamp in seconds.
+        seconds: The Harp timestamp data, in fractional seconds, to convert to datetime.
 
     Returns:
-        datetime.datetime, pandas.Series: The Harp timestamp(s) in datetime format.
+        datetime: If parsing succeeded. Return type depends on input.
+          - scalar: datetime object representing the decoded UTC Harp timestamp
+          - Index: DatetimeIndex of datetime64 dtype
+          - Series: Series of datetime64 dtype
```

Comment:
In the Returns, the part ("datetime" here) before the `:` is treated as the return type, which gives us:
<img width="486" height="326" alt="image" src="https://github.com/user-attachments/assets/7f433c13-4129-4423-9c71-356ddd8555e6" />

To get what we want:
<img width="481" height="320" alt="image" src="https://github.com/user-attachments/assets/a913f0a0-f048-4805-a1e2-95ee4a0481ae" />

There should also be a blank line before the start of a list, with items aligned to the first line of the return description
```suggestion
        The decoded UTC Harp timestamp(s). Return type depends on input.

        - scalar: datetime object
        - Index: DatetimeIndex of datetime64 dtype
        - Series: Series of datetime64 dtype
```